app/src/main/res/layout/fragment_create_game_form.xml

```diff
@@ -0,0 +1,306 @@
+        </LinearLayout>
+
+        <!-- Difficulty -->
+        <LinearLayout
+            android:id="@+id/layout_difficulty"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:paddingLeft="8dp"
+            android:paddingRight="8dp"
+            app:layout_constraintTop_toBottomOf="@+id/layout_numb_players">
+
+            <TextView
+                android:id="@+id/difficulty"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/difficulty"
+                app:layout_constraintLeft_toLeftOf="parent" />
+
+            <Spinner
+                android:id="@+id/difficulty_spinner"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:entries="@array/difficulties_array"
+                app:layout_constraintBaseline_toBaselineOf="@id/difficulty"
+                app:layout_constraintLeft_toRightOf="@+id/difficulty"
+                app:layout_constraintRight_toRightOf="parent" />
+
+        </LinearLayout>
+
+        <!-- Universe -->
+        <LinearLayout
+            android:id="@+id/layout_universe"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:paddingLeft="8dp"
+            android:paddingRight="8dp"
+            app:layout_constraintTop_toBottomOf="@+id/layout_difficulty">
+
+            <TextView
+                android:id="@+id/universe"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/universe"
+                app:layout_constraintLeft_toLeftOf="parent" />
+
+            <Spinner
+                android:id="@+id/universes_spinner"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="8dp"
+                android:entries="@array/universes_array"
+                app:layout_constraintBaseline_toBaselineOf="@id/universe"
+                app:layout_constraintLeft_toRightOf="@+id/universe" />
+
+
+            <EditText
+                android:id="@+id/universe_field"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="8dp"
+                android:layout_weight="1"
+                android:inputType="textPersonName"
+                android:maxLength="30"
+                app:layout_constraintBaseline_toBaselineOf="@id/universe"
+                app:layout_constraintLeft_toRightOf="@id/universes_spinner"
+                app:layout_constraintRight_toRightOf="parent" />
+        </LinearLayout>
+
+        <!--TODO-->
```

Comment:
By whom? Please use `TODO(@DonnA98)` so we can trace back who's responsible for doing this

Reply:
My bad that was for me